packages/storybook/layouts.stories.js

```diff
@@ -113,54 +113,88 @@ storiesOf('Layouts', module)
       </cxl-layout-marketing>
     `;
   })
-  .add('cxl-layout-institute', () => {
+  .add('cxl-institute-layout', () => {
     return html`
-      <cxl-layout-institute>
-        <div class="wrapper">
-          <div id="left-nav">
-            <a href="/">
-              <div class="logo">
-                <span class="cxl">C<span>XL</span></span>
-              </div>
-            </a>
-            <div class="top-nav">
-              <ul class="nav-items">
-                <li>
-                  <a href="trainings" class="active">
-                    <iron-icon class="size-l" icon="cxl:nav-trainings"></iron-icon>
-                    <iron-icon class="size-l" icon="cxl:nav-trainings-active"></iron-icon>
-                    <span>Trainings</span>
-                  </a>
-                </li>
-                <li>
-                  <a href="roadmap">
-                    <iron-icon class="size-l" icon="cxl:nav-roadmap"></iron-icon>
-                    <iron-icon class="size-l" icon="cxl:nav-roadmap-active"></iron-icon>
-                    <span>Discover</span>
-                  </a>
-                </li>
-                <li>
-                  <a href="discover">
-                    <iron-icon class="size-l" icon="cxl:nav-discover"></iron-icon>
-                    <iron-icon class="size-l" icon="cxl:nav-discover-active"></iron-icon>
-                    <span>Manage</span>
-                  </a>
-                </li>
-              </ul>
-            </div>
-            <div class="bottom-nav">
-              <ul class="nav-items">
-                <li>
-                  <a href="/">
-                    <iron-icon class="size-l" icon="lumo:user"></iron-icon>
-                    <span>Profile</span>
-                  </a>
-                </li>
-              </ul>
+      <div class="wrapper">
+        <div id="left-nav">
+          <a href="/">
+            <div class="logo">
+              <span class="cxl">C<span>XL</span></span>
             </div>
+          </a>
+          <div class="top-nav">
+            <ul class="nav-items">
```

Comment:
Icon nav will highly likely be modelled in WP menu, so follow all the `nav > ul > li > a` structure, with respective WP class naming conventions.

For this specific icon menu, we will make WP menu item label the icon ID (like `cxl:nav-trainings`), and filter PHP output to create icon tags, etc.